### DISCUSSION TITLE
GY-91 board with MPU9250 not work

### DISCUSSION THREAD

**Discussion by skimans** [iNavFlight/inav#8617] [General]
Hi
many GY-91 board with MPU9250 not work.
It's strange this...
i dont know why this..
do you have experience with GY-91 and you have solved this problem ?

**Comment by erstec** [comment]
@skimans there is nothing strange, as you dont tell us how and to what MCU you connect it, how your target definitions looks like and all other details needed to understand what you are doing.
GY-91 works as expected (at least on my bench, not for real flights ofc, as I am adherent of using proper FC for real flights, but all those DIY breakout boards is ok for testings and development) if it is connected properly (via SPI) and target are created.
